recipes/pyg4ometry/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "pyg4ometry" %}
+{% set version = "1.0.1" %}
+{% set antlr_version = "4.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://bitbucket.org/jairhul/pyg4ometry/get/v{{ version }}.tar.gz
+    sha256: 90372cefdaafcc3101a3122ade18b684d5aa725998fb5cc03d7c411267312a9b
+    folder: pyg4ometry
+    patches:
+      - 0001-Fix-setting-includeSearchDirs-and-librarySearchDirs.patch
+      - 0002-Remove-test-only-dependencies.patch
+  - url: https://www.antlr.org/download/antlr-{{ antlr_version }}-complete.jar
```

Comment:
Not sure if our package would suffice here. Did you try it?

Reply:
I swear I looked for that and didn't find it 😆 

Though it's probably easier not to use it as it's build-time-only, needs this specific version and I'm interested in building for `osx-arm64`.